flocker/restapi/docs/publicapi.py

```diff
@@ -183,6 +183,8 @@ def _introspectRoute(route, exampleByIdentifier, schema_store):
     """
     result = {}
 
+    result['header'] = route.attributes.get('header', 'No header!')
```

Comment:
Another option would be to raise [`SphinxError`](http://sphinx-doc.org/extdev/appapi.html#sphinx.errors.SphinxError) here.